scripts/table_updates/tests/test_utilities.py

```diff
@@ -313,21 +349,30 @@ def get__update_tier1a_data_replacement_mapping_table_test_cases():
 def test_update_tier1a_data_replacement_mapping_table(syn, table_schema, test_cases, config):
     with patch.object(syn, "get", return_value=table_schema) as patch_get, patch.object(
         syn, "tableQuery"
-    ) as patch_table_query, patch.object(syn, "store") as patch_store, patch.object(syn, "delete") as patch_delete:
+    ) as patch_table_query, patch.object(syn, "store") as patch_store, patch.object(syn, "delete") as patch_delete, patch('utilities.update_version') as update_version:
         patch_table_query.return_value = MagicMock(etag="test_etag")
+        logger = MagicMock(spec=logging.Logger)
+
+        comment = "test comment"
         # Call the function
         update_tier1a_data_replacement_mapping_table(
```

Comment:
I think tests for this function `update_tier1a_data_replacement_mapping_table` needs to be updated to include testing for the new parameters: `bpc_column_list, `main_genie_column_list`.

Reply:
Sure. I will tweak this.